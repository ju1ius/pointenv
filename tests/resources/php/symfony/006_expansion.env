FOO=foo

A=$FOO
B="$FOO"
C='$FOO'

D=${FOO}bar
E="${FOO}bar"
F='${FOO}bar'

G=$FOO}bar
H="$FOO}bar"
I='$FOO}bar'

J=\$FOO
K="\$FOO"
L='\$FOO'
M=\${FOO]
N="\${FOO}"
O='\${FOO}'
