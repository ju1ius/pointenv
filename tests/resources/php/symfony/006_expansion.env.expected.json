{
  "FOO": "foo",
  "A": "foo",
  "B": "foo",
  "C": "$FOO",
  "D": "foobar",
  "E": "foobar",
  "F": "${FOO}bar",
  "G": "foo}bar",
  "H": "foo}bar",
  "I": "$FOO}bar",
  "J": "$FOO",
  "K": "$FOO",
  "L": "\\$FOO",
  "M": "${FOO]",
  "N": "${FOO}",
  "O": "\\${FOO}",
  "P1": "$~",
  "P2": "${666}",
  "Q1": "f\\no",
  "Q2": "f\no"
}
